frontend/src/locales/en.json

```diff
@@ -357,6 +357,21 @@
     "shareMyLink": "Share my link",
     "reportBug": "Report Bug"
   },
+  "notices": {
+    "betaWarning": {
+      "heading": "As we're currently in beta, we do have some issues we're dealing with:",
+      "list": [
+        "Please check your spam folder as some emails might end up there",
+        "If Zoom meetings don't appear in your appointment invite, please disconnect and reconnect your zoom account in the {connectedAccounts} section of the settings",
```

Comment:
Falls Zoom Meetings nicht in der Kalender-Einladung auftauchen, bitte trenne die Verbindung im {connectedAccounts} Abschnitt der Einstellungen, und stelle sie erneut her